.tools/test/stacks/plugin/typescript/lib/plugin_stack.ts

```diff
@@ -0,0 +1,285 @@
+    this.initSqsLambdaIntegration(batchFunction, sqsQueue);
+    if (acctConfig[`${toolName}`].status === "enabled") {
+      this.initLogFunction(adminBucketName);
+    }
+  }
+
+  private getYamlConfig(filepath: string): any {
+    return yaml.load(fs.readFileSync(filepath, 'utf8'));
+  }
+
+  private initGetTopic(topicName: string): sns.ITopic {
+    const externalSnsTopicArn = `arn:aws:sns:${this.awsRegion}:${this.adminAccountId}:${topicName}`;
+    return sns.Topic.fromTopicArn(this, 'ExternalSNSTopic', externalSnsTopicArn);
+  }
+
+  private initBatchFargate(): [batch.CfnJobDefinition, batch.CfnJobQueue] {
+    const batchExecutionRole = new iam.Role(this, `BatchExecutionRole-${toolName}`, {
+      assumedBy: new iam.ServicePrincipal('ecs-tasks.amazonaws.com'),
+      roleName: `BatchExecutionRole-${toolName}`,
+      inlinePolicies: {
+        BatchLoggingPolicy: new iam.PolicyDocument({
+          statements: [
+            new iam.PolicyStatement({
+              effect: iam.Effect.ALLOW,
+              actions: [
+                'logs:CreateLogGroup',
+                'logs:CreateLogStream',
+                'logs:PutLogEvents',
+                'logs:DescribeLogStreams',
+              ],
+              resources: ['arn:aws:logs:*:*:*'],
+            }),
+            // new iam.PolicyStatement({
+            //   effect: iam.Effect.ALLOW,
+            //   actions: [
+            //     'ecr:*',
+            //   ],
+            //   resources: ['arn:aws:*:*:*:*'],
+            // }),
+          ],
+        }),
+      },
+      managedPolicies: [
+        iam.ManagedPolicy.fromAwsManagedPolicyName('AdministratorAccess'),
+        iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonEC2ContainerRegistryReadOnly'),
+        iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AmazonECSTaskExecutionRolePolicy'),
+      ],
+    });
+
+    const vpc = ec2.Vpc.fromLookup(this, 'Vpc', { isDefault: true });
+    const sg = new ec2.SecurityGroup(this, "sg", {
+            securityGroupName: "batch-sg",
+            vpc
+    });
+    const fargateEnvironment = new batch.CfnComputeEnvironment(this, `FargateEnv-${toolName}`, {
+      type: "MANAGED",
+      computeResources: {
+        type: "FARGATE",
+        subnets: vpc.selectSubnets().subnetIds,
+        securityGroupIds: [sg.securityGroupId],
+        maxvCpus: 1,
+      }
+    });
+
+    const containerImageUri = `${this.adminAccountId}.dkr.ecr.us-east-1.amazonaws.com/${toolName}:latest`;
+
+    const jobDefinition = new batch.CfnJobDefinition(this, 'JobDefn', {
+      type: "container",
+      containerProperties: {
+        image: containerImageUri,
+        jobRoleArn: batchExecutionRole.roleArn,
+        executionRoleArn: batchExecutionRole.roleArn,
+        networkConfiguration: {
+          assignPublicIp: 'ENABLED',
+        },
+        resourceRequirements: [
+          {
+            type: "VCPU",
+            value: "0.25"
```

Comment:
This feels both small and like something that we would want to configure

Reply:
Added this.batch_memory with ?? 16GB if it's not set, same for VCPU's which will default to 4.